docs/data/charts/components/SeriesDemo.js

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { ChartsXAxis } from '@mui/x-charts/ChartsXAxis';
+import { ChartsYAxis } from '@mui/x-charts/ChartsYAxis';
+import { LinePlot } from '@mui/x-charts/LineChart';
+import { useLineSeries, useXAxis, useXScale, useYScale } from '@mui/x-charts/hooks';
+import { ChartsSurface } from '@mui/x-charts/ChartsSurface';
+import { ChartDataProvider } from '@mui/x-charts/ChartDataProvider';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+
+function ExtremaLabels() {
+  const lineSeries = useLineSeries();
+
+  if (!lineSeries) {
+    return null;
+  }
+
+  return (
+    <React.Fragment>
+      {lineSeries.seriesOrder.map((seriesId) => (
+        <SingleSeriesExtremaLabels series={lineSeries.series[seriesId]} />
+      ))}
+    </React.Fragment>
+  );
+}
+
+function SingleSeriesExtremaLabels({ series }) {
+  const xAxis = useXAxis('x');
+
+  const min = series.data.reduce(
+    (acc, value) => Math.min(acc ?? Infinity, value ?? Infinity),
+    Infinity,
+  );
+  const max = series.data.reduce(
+    (acc, value) => Math.max(acc ?? -Infinity, value ?? -Infinity),
+    -Infinity,
+  );
+
```

Comment:
Sorry for second phase of feedback.
This other solution should give you an object with the index and the value of the min point.

```
  const minPoint = series.data.reduce(
    (currentPoint, value, index) => value < acc.minValue ? { index, value } : currentPoint,
    {index: -1, value: Infinity },
  );
  
```

The goal is to then be able to do something like that instead of looping on the points  a second time

```
<PointLabel
            x={xAxis.data?.[minPoint.index]}
            y={minPoint.value}
            placement='below'
            color={series.color}
          />
```

Reply:
+1 for simplifying the data. The purpose is to demonstrate the `useSeries` not to show how to compute min/max ;)

Reply:
Yeah, good point! Done 👍